crates/cxx-qt-lib/src/core/qcommandlineparser.cpp

```diff
@@ -0,0 +1,49 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#include "cxx-qt-lib/qcommandlineparser.h"
+
+#include "../assertion_utils.h"
+
+// QCommandLineParser has a single pointer as it's member
+//
+// https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/tools/qcommandlineparser.h?h=v5.15.6-lts-lgpl#n107
+// https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/tools/qcommandlineparser.h?h=v6.2.4#n109
+assert_alignment_and_size(QCommandLineParser,
+                          alignof(::std::size_t),
+                          sizeof(::std::size_t));
+
+static_assert(!::std::is_trivially_copy_assignable<QCommandLineParser>::value);
+static_assert(
```

Comment:
did you intentionally skip implementing Clone in Rust ? eg do we want to be able to copy this in Rust?